scripts/rebuild-icds-staging

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
-
+echo ""
+echo "This command will be deprecated soon. Instead use the tool in commcare-cloud:"
+echo "    $ python scripts/rebuild-deploy-branches icds-staging --commcare-hq-root=/path/to/commcare/repo/"
```

Comment:
nit: for people to not figure this out when they switch, can we already put what the commcare hq root path would be for icds-staging? Also it's not clear why one would need to mention this for rebuilding staging branch but I think I will understand that better when the command is there.

Reply:
The command is already there. The path is local to your machine so I can't put it in there, it depends on where you cloned it to on your machine.

Reply:
ah i see. It was not clear to me that it was to be the local repo. Thanks
nit: then that probably should be `/path/to/commcare-hq/repo`